### ISSUE TITLE
Add session management

### ISSUE THREAD

**Issue by marc-kdab** [KDAB/Charm#138]
Tested with v1.8.0 on Linux:
1. Start a task
2. Log out
3. Log in

-> Charm is started, but does not continue tracking on the selected task.

It's much easier to edit data away than it is to add, so the tracking should continue, preferably within a separate event of the same task, so as to record the timespan the user was logged out.

**Comment by frankosterfeld** [comment]
I don't think we should just continue tracking, but it could work like idle detection, asking after login if one wants to end the event at logout time or continue tracking.

**Comment by pasnox** [comment]
While there, pelase also add support for restoring openedcharm windows (like, month and week summary).
